### PR TITLE
chore(clowdapp): add mosquitto MQTT to opt deps

### PR DESCRIPTION
**Pull request by vkrizan**
## Secure Coding Practices Checklist GitHub Link
    - https://github.com/RedHatInsights/secure-coding-checklist

    ## Secure Coding Checklist
    - [ ] Input Validation
    - [ ] Output Encoding
    - [ ] Authentication and Password Management
    - [ ] Session Management
    - [ ] Access Control
    - [ ] Cryptographic Practices
    - [ ] Error Handling and Logging
    - [ ] Data Protection
    - [ ] Communication Security
    - [ ] System Configuration
    - [ ] Database Security
    - [ ] File Management
    - [ ] Memory Management
    - [ ] General Coding Practices

### CODE REVIEW

**Review by rexwhite — APPROVED** [review]
'Mosquitto' 😂...

File: deployment/clowdapp.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ objects:
     - compliance-ssg
     - config-manager
     - cloud-connector
+    # add MQTT mock (mosquitto) from cloud-connector
+    - mosquitto
```
Comment (vkrizan) [inline deployment/clowdapp.yml:22]: We need to check whether this wouldn't cause issues on stage/prod. Event cloud-connector does not lists this as an (optional) dependency, as it is used only for testing on ephemeral envs.

We can follow the same, but then the cloud-connector needs to be deployed explicitly, as `mosquitto` is pulled out of possible components of the "cloud-connector" app (as per app-interface definitions).

Note, that these lists components and not app names.

**Comment by vkrizan** [comment]
@rexwhite now I've got the pun :joy:

File: deployment/clowdapp.yml (hunk 1 above)
Comment (rexwhite) [reply deployment/clowdapp.yml:22]: Yes - let's not break prod, please 😬...

**Pull request merged**